packages/core/src/components/callout/_callout.scss

```diff
@@ -25,20 +25,20 @@ Styleguide callout
   @include running-typography();
   background-color: rgba($gray3, 0.15);
   border-radius: $pt-border-radius;
-  padding: $pt-grid-size ($pt-grid-size * 1.2) ($pt-grid-size * 0.9);
+  padding: $pt-grid-size * 1.5;
   position: relative;
   width: 100%;
 
   // CSS API support
   &[class*="#{$ns}-icon-"] {
-    padding-left: ($pt-grid-size * 2) + $pt-icon-size-large;
+    padding-left: ($pt-grid-size * 2.2) + $pt-icon-size-standard;
```

Comment:
2.2 is 1.5 (consistent side padding) and 0.7 for spacing between icon and text (consistent with buttons).